.gitignore

```diff
@@ -204,4 +202,4 @@ $RECYCLE.BIN/
 # End of https://www.toptal.com/developers/gitignore/api/node,visualstudiocode,windows,linux
 
 # key.js (API_KEY)
-src/key.js
```

Comment:
If we are storing their API key inside .env file, and the key.js has been removed, this should no be needed, no?